### PR TITLE
Katherine Surma

### PR DESCRIPTION
**Pull request by kmsurma**
After forking and cloning the repository to my computer, I opened the README.md file in RStudio and made changes to it, including adding headings, italics, images, bullet points, links, and gifs. 

The directions I used for implementing these changes were all found on one website, which can be found here: https://rmarkdown.rstudio.com/authoring_basics.html. I was able to create headings by including three pound signs before the text I wanted to be in the headers. I was able to italicize text by placing an asterisk before and after the relevant text. I made the bullet point list by placing one asterisk before each item, and placing each item on a separate line. I was able to add gifs and images to the file by saving them to my computer and placing them in the document using the following code: . Lastly, I was able to add links with the following code: [text that I want to be linked](web address).

For me, implementing the changes in RStudio was the easiest part of the assignment, while transferring my file onto GitHub was the hardest. This is because I have never used GitHub before, while I have had a little exposure to R Markdown. I ran into a coding problem when I tried to embed images and gifs in the assignment. I initially thought the brackets that must be included in the code were option if you did not want to put a caption on a picture, but I discovered that they are required: if you don't want a caption on your pictures, you simply leave them empty. I also had some initial trouble creating a pull request because I attempted to do so without first creating a branch that contained my commits. After committing my changes, however, the process of creating a pull request went smoothly. However, I am unsure if I made the changes to the correct "branch". I found the instructions on this part to be a bit confusing, since they noted that changes should be proposed to a "branch" so that the "master branch" only contains approved work. I was unsure which "branch" I should select to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
-# hw01
-Repository for homework 1
+# HW01
+## Katherine M. Surma
+### About Me
+
+I am a joint-degree BA/MA student at the University of Chicago. I am earning my BA in Political Science and Law, Letters, & Society, and I am completing my Master's degree through the [Master of Arts Program in the Social Sciences](https://mapss.uchicago.edu/node).
```
Comment (soohyun1995) [inline README.md:5]: Good job on putting the link!!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+### Career Interests
+
+I *hope* to attend law school to become a federal prosecutor. 
+
+![](elle.gif)
+
+
+
+*However*, I have really enjoyed my time in MAPSS and am therefore also considering pursuing a PhD in Political Science.
```
Comment (soohyun1995) [inline README.md:23]: I love how you incorporated words in italic in each sentence!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+
+![](elle.gif)
+
+
+
+*However*, I have really enjoyed my time in MAPSS and am therefore also considering pursuing a PhD in Political Science. 
+
+![](sheldon.gif)
+
+### Personal Life
```
Comment (soohyun1995) [inline README.md:27]: Great job on using headers!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+## Katherine M. Surma
+### About Me
+
+I am a joint-degree BA/MA student at the University of Chicago. I am earning my BA in Political Science and Law, Letters, & Society, and I am completing my Master's degree through the [Master of Arts Program in the Social Sciences](https://mapss.uchicago.edu/node).  
+
+![I'm on the right in the above picture.](katherine.jpg)
+
+### Research Interests
+
+* American politics
```
Comment (soohyun1995) [inline README.md:11]: Great job on using lists!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,41 @@
...
+* Misinformation 
+* Voter Preference
+
+### Career Interests
+
+I *hope* to attend law school to become a federal prosecutor. 
+
+![](elle.gif)
```
Comment (soohyun1995) [inline README.md:19]: I love how you also put gif in!

**Review by soohyun1995 — COMMENTED** [review]
The coding style, presentation, and creativity were all excellent! I liked how you incorporated headers, italics, lists, photos, and links very well to your biography!

File: README.md (hunk 4 above)
Comment (andrewchill25) [inline README.md:11]: The lists look great in both the the code and its rendering! Also the picture is something I couldn't figure out. I had overlooked the required (!)

File: README.md (hunk 1 above)
Comment (andrewchill25) [inline README.md:5]: Links gave me some trouble. I wasn't paying enough attention.

**Review by andrewchill25 — COMMENTED** [review]
Everything looked great! Your comments about your process were especially helpful. They were very clear and comprehensive. 
Great work!

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for Katherine Surma
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

This is a really good job for the first assignment.  You include a number of different elements, multiple photos and gifs, and do a thorough job reflecting on the experience and the challenges.  You also used an efficient way to link the photos by saving them in the repo and pushing them to git (instead of just linking to websites they are on).

**Pull request closed without merging**